examples/step-71/doc/intro.dox

```diff
@@ -0,0 +1,678 @@
+function* for such a material is only parameterized in terms of the
+(current) field variables, but not their time derivatives or past values.
+
+We'll choose the energy density function, which captures both the energy
+stored in the material due to deformation and magnetization, as well as
+the energy stored in the magnetic field itself, to be
+@f[
+  \psi_{0} \left( \mathbf{C}, \boldsymbol{\mathbb{H}} \right)
+= \frac{1}{2} \mu_{e} f_{\mu_{e}} \left( \boldsymbol{\mathbb{H}} \right)
+    \left[ \text{tr}(\mathbf{C}) - d - 2 \ln (\text{det}(\mathbf{F}))
+    \right]
++ \lambda_{e} \ln^{2} \left(\text{det}(\mathbf{F}) \right)
+- \frac{1}{2} \mu_{0} \mu_{r} \text{det}(\mathbf{F})
+    \left[ \boldsymbol{\mathbb{H}} \cdot \mathbf{C}^{-1} \cdot
+    \boldsymbol{\mathbb{H}} \right]
+@f]
+with
+@f[
+  f_{\mu_{e}} \left( \boldsymbol{\mathbb{H}} \right)
+= 1 + \left[ \frac{\mu_{e}^{\infty}}{\mu_{e}} - 1 \right]
+    \tanh \left( 2 \frac{\boldsymbol{\mathbb{H}} \cdot
+    \boldsymbol{\mathbb{H}}}
+      {\left(h_{e}^{\text{sat}}\right)^{2}} \right)
+@f]
+and for which the variable $d = \text{tr}(\mathbf{I})$ ($\mathbf{I}$
+being the rank-2 identity tensor) represents the spatial dimension and
+$\mathbf{F}$ is the deformation gradient tensor. To give some brief
+background to the various components of $\psi_{0}$, the first two terms
+bear a great resemblance to the stored energy density function for a
+(hyperelastic) Neohookean material. The only difference between what's
+used here and the Neohookean material is the scaling of the elastic shear
+modulus by the magnetic field-sensitive saturation function $f_{\mu_{e}}
+\left( \boldsymbol{\mathbb{H}} \right)$ (see @cite Pelteret2018a, equation
+29). This function will, in effect, cause the material to stiffen in the
+presence of a strong magnetic field. As it is governed by a sigmoid-type
+function, the shear modulus will asymptotically converge on the specified
+saturation shear modulus. It can also be shown that the last term in
+$\psi_{0}$ is the stored energy density function for magnetic field (as
+derived from first principles), scaled by the relative permeability
+constant. This definition collectively implies that the material is
+linearly magnetized, i.e., the magnetization vector and magnetic field
+vector are aligned. (This is certainly not obvious with the magnetic energy
+stated in its current form, but when the magnetic induction and magnetization
+are derived from $\psi_{0}$ and all magnetic fields are expressed in the
+<em>current configuration</em> then this correlation becomes clear.)
+As for the specifics of what the magnetic induction, stress tensor, and the
+various material tangents look like, we'll defer presenting these to the
+tutorial body where the full, unassisted implementation of the constitutive
+law is defined.
+
+<h5>Magneto-viscoelastic constitutive law</h5>
+
+The second material that we'll formulate is one that for a
```

Comment:
The second material that we'll formulate is one for a

I think this is what you mean here